index.html

```diff
@@ -136,6 +136,164 @@ <h2>
+        input control that can be pressed to activate. A [=gamepad=] has a
+        <dfn>button list</dfn> which is a [=list=] containing all the
+        [=button=] inputs for the [=gamepad=] in some order determined by the
+        [=user agent=].
+      </p>
+      <p>
+        An input control may be designed to automatically return a [=button=]
+        to an unpressed state when the user stops interacting with the input
+        control. If so, the [=button=] has a <dfn>preferred button state</dfn>.
+      </p>
+      <p>
+        A [=button=] may have a digital switch to indicate when the [=button=]
+        is activated. If so, the [=button=] has a <dfn>digital button
+        value</dfn> [=input value=] that is `true` when the [=button=] is
+        activated and `false` otherwise.
+      </p>
+      <p>
+        A [=button=] may have an analog sensor enabling the [=button=] to
+        report the degree to which the [=button=] is activated. If so, the
+        [=button=] has these [=input values=]:
+      </p>
+      <ul>
+        <li>An <dfn>analog button value</dfn>, an integer value that represents
+        the degree of activation.
+        </li>
+        <li>A <dfn>minimum analog value</dfn> representing no activation. The
+        smallest in-bounds [=analog button value=].
+        </li>
+        <li>A <dfn>maximum analog value</dfn> representing full activation. The
+        largest in-bounds [=analog button value=].
+        </li>
+      </ul>
+      <p>
+        A [=button=] may be capable of detecting touch. If so, the [=button=]
+        has a <dfn>button touch value</dfn> [=input value=] that is `true` when
+        the [=button=] is touched and `false` otherwise.
+      </p>
+      <p>
+        A [=gamepad=] may have [=touch surfaces=]. A <dfn>touch surface</dfn>
+        is an input control that provides 2D position data representing points
+        of contact. A [=gamepad=] has a <dfn>touch surface list</dfn> which is
+        a [=list=] containing the [=touch surfaces=] for the [=gamepad=]. The
+        list is ordered such that [=touch surfaces=] closer to the left side of
+        the [=gamepad=] appear closer to the start of the list.
+      </p>
+      <p>
+        A [=touch surface=] has an <dfn>active touch point list</dfn> [=input
+        value=], a [=list=] of zero or more [=touch points=] representing the
+        points of contact currently detected by the sensor, ordered such that
+        earlier [=touch points=] appear closer to the start of the [=list=]. A
+        <dfn>touch point</dfn> represents a single point of contact at a single
+        point in time. A [=touch point=] has a <dfn>touch x coordinate</dfn>
+        representing its position along the user's left-right axis and a
+        <dfn>touch y coordinate</dfn> representing its position along the
+        perpendicular axis.
+      </p>
+      <p>
+        A [=touch surface=] may have surface dimension [=input values=]. The
```

Comment:
Are the surface dimensions considered `input values`? According to our definition, `input values` are associated with `input controls`. Maybe we could just say that dimensions are just "values"?

Reply:
I went back and forth on this. In general I want to divide gamepad properties into "read when the gamepad is first connected and never changes" and "read each time inputs are read from the gamepad". We don't expect touch surface dimensions to change since they're supposed to represent the dimensions of the hardware touch sensor. But we never require that they *can't* change, so it seems safer to treat them as if they could change. What if it's a virtual gamepad that lets the user resize the touch surface area?